repository/templates/base.html

```diff
@@ -41,12 +41,12 @@
 
     <header class="site-header">
       <div class="container">
-          <a href="/" class="site-title"><h1>mySociety Research</h1></a>
+          <a href="{% url 'home' %}" class="site-title"><h1>mySociety Research</h1></a>
           <nav role="navigation" class="site-nav">
               <ul>
-                  <li><a href="{% url 'home' %}">Home</a></li>
-                  <li><a href="{% url 'page' 'strategy' %}">Research Strategy</a></li>
-                  <li><a href="{% url 'items' %}">Our Research</a></li>
+                  <li><a href="{% url 'items' %}">Publications</a></li>
+                  <li><a href="https://www.mysociety.org/category/research/">Blog</a></li>
```

Comment:
Might be worth checking with designers on if they have any views/thoughts on pushing people off-site in top-level navigation.